tasks/deploy.yaml

```diff
@@ -98,9 +98,9 @@ spec:
 
         echo "deploying to ephemeral"
         if [ -z "${EXTRA_DEPLOY_ARGS}" ]; then
-          EXTRA_DEPLOY_ARGS="$(parse-snapshot.py)"
+          EXTRA_DEPLOY_ARGS=" $(parse-snapshot.py) "
```

Comment:
I don't think this is needed here since it is checking that the `EXTRA_DEPLOY_ARGS` is empty